blog/content/post/2025-11-03-nis2-digital-sovereignty-with-zarhus.md

```diff
@@ -0,0 +1,327 @@
+exploited to deploy bootkits, showcasing the reliance on vendor key handling
+and revocations." style="border: 2px solid grey;">
+<center>
+<i>
+Source: <a
+href="https://web.archive.org/web/20250116151152/https://www.bleepingcomputer.com/news/security/new-uefi-secure-boot-flaw-exposes-systems-to-bootkits-patch-now/">bleepingcomputer.com</a>
+</i>
+</center>
+<br><br>
+
+Past vulnerabilities have shown how attackers can exploit insecure firmware to
+inject malware or subvert boot processes, rendering traditional defenses
+ineffective. Furthermore, the complexity of proper cryptographic key handling
+and revocations adds to these risks, emphasizing the need for stronger
+sovereignty over firmware security mechanisms.
+
+According to the [2025 Cyber Threat Analysis by
+IDS-INDATA](https://web.archive.org/web/20251031110741/https://idsindata.co.uk/20-rise-in-supply-chain-cyberattacks-legacy-systems-continue-to-leave-manufacturing-exposed/),
+supply chain attacks have risen by 20% between 2024 and 2025. This increase
+identifies them as the fastest-growing cybersecurity threat.
+
+![A chart showcasing the percentage difference between various kinds of cyber
+attacks, supply chain attacks ranking the
+highest](/img/nis2-with-zarhus/supply_chain_attacks_2024_2025.png)
+
+## Taking Control: Building Your Firmware Security Infrastructure
+
+Achieving true digital sovereignty over firmware requires owning a secure
+infrastructure for the retrieval and provisioning of open source code. This
+entails customizing the sources so that they are tailored precisely to the
+organization’s machines and establishing a secure compartment for signing
+firmware binaries. Although setting up such infrastructure may initially seem
+daunting and resource-intensive, advances in dedicated hardware and automation
+tools now make this approach practical and sustainable.
+
+A pivotal innovation in this domain is the advent of dedicated,
+secure-by-design hardware for automation.
+
+Enter: **Zarhus Trust Root Provisioning Box**.
+
+![Zarhus Trust Root Provisioning Box](/img/nis2-with-zarhus/zpb.jpg)
+
+This device empowers organizations to generate and securely store cryptographic
+keys in an encrypted manner on physical media, such as USB flash drives, where
+they remain under sole control. It also enables the fortification of the boot
+process via mechanisms such as Intel Boot Guard and UEFI Secure Boot, but with
+keys managed exclusively by the organization, rather than third-party vendors.
+
+This means that your organization can automate the secure signing of the
+firmware tailored to your machines and provision devices confidently, knowing
+that you control the entire trust chain from key generation to deployment.
+
+## Example key generation
+
+Operating this hardware solution is straightforward, with guided command-line
+interfaces facilitating key creation, firmware signing, and Secure Boot key
+enrollment, catalyzing productivity without compromising security.
```

Comment:
We should tell more comprehensive story why someone in context of NIS2 would like to have own UEFI Secure Boot keys and what it means for system maintanance and IT administration.

Reply:
fixed: https://github.com/3mdeb/news-and-ideas/compare/880ebe25791e5a07ce3786c7f6ad93c125317f2f..69a5ce00389117bf576d5ac9648e1545fd661a12

Reply:
On one side, it is extremely verbose, but on the other side, I like the narrative and the way you think here. Approved.